server/schemas/property.schema.js

```diff
@@ -7,11 +7,18 @@ const mapLocation = Joi.object().keys({
   latitude: Joi.string().label('Map location latitude').optional(),
 });
 
-const product = {
+const address = Joi.object().keys({
+  street1: Joi.string().label('Property Street 1').optional(),
```

Comment:
This should be compulsory. Also, I dont think we need the Property prefix.
```suggestion
  street1: Joi.string().label('Street 1').required(),
```